e2e-test/wds_azure_e2etest.py

```diff
@@ -31,21 +31,24 @@ def run_workspace_app_test(cbas, wds_upload, cbas_submit_workflow, test_cloning,
     else:
         print("TEST COMPLETE.")
 
+    upload_success = False
+
     if wds_upload:
         logging.info(f"trying to see wds is ready to upload to workspace {workspace_id}")
         wds_url = poll_for_app_url(workspace_id, "WDS", "wds", azure_token, leo_url)
         if wds_url == "":
             logging.error(f"wds errored out for workspace {workspace_id}")
         else:
-            upload_wds_data(wds_url, workspace_id, "resources/test.tsv", "test", azure_token)
+            upload_success = upload_wds_data(wds_url, workspace_id, "resources/test.tsv", "test", azure_token)
 
     if cbas_submit_workflow:
         # next trigger a workflow in each of the workspaces, at this time this doesnt monitor if this was succesful or not
         # upload file needed for workflow to run
-        upload_wds_data(wds_url, workspace_id, "resources/sraloadtest.tsv", "sraloadtest", azure_token)
+        upload_success = upload_wds_data(wds_url, workspace_id, "resources/sraloadtest.tsv", "sraloadtest", azure_token)
         submit_workflow_assemble_refbased(workspace_id, "resources/assemble_refbased.json", azure_token)
 

```

Comment:
since you removed the assert on `response.records_modified == count` in helper.py, should you put it back in here, asserting on `upload_success`?

Reply:
I don't think I actually removed it - although since at the moment we actually run this testing using `python` instead of `pytest`, I'm not sure the assert actually does anything (if it did, presumably the test would fail at that point and I wouldn't need to stop the clone from going forward?).  So maybe I should remove it, unless we want to start using `pytest` (I think I switched to `python` due to version incompatibilities that were too frustrating and difficult to resolve).

Reply:
oh duh, my bad, I misread the diff - ignore that feedback! But yeah, if the assert does nothing maybe you could change it to manually throw an error or something.